README.md

```diff
@@ -12,6 +12,8 @@ Rebar3/Rebar plugin for [asdf version manager](https://github.com/HashNuke/asdf)
 
 ```
 asdf plugin-add rebar https://github.com/Stratus3D/asdf-rebar.git
+
```

Comment:
Thanks for the PR, however the `Install` heading was intended to only contain instructions for install this asdf-rebar plugin, not using the plugin to install rebar.